Readme.md

```diff
@@ -6,4 +6,22 @@
 
 # How to load Scheduler Control data on demand
 
-This example illustrates how to load data in the scheduler only for the selected date interval.
+This example loads data in the Scheduler only for the selected date interval. The example consists of three parts.
+
+* _CommonDbContext_. This solution uses a single `DbContext` for the application and has own SchedulingViewModel.
+* _DbContextPerRequest_. This solution follows the One `DbContext` per request approach and has own SchedulingViewModel.
+* _Shared_. The shared project contains the data and the view, which are the same for both solutions. The events are mapped to the view model’s handlers.
+
+## Files to Review
+
+- [SchedulingViewModel.cs](./CS/CommonDbContext/ViewModels/SchedulingViewModel.cs) ([SchedulingViewModel.vb](./VB/CommonDbContext/ViewModels/SchedulingViewModel.vb))
+- [SchedulingViewModel.cs](./CS/DbContextPerRequest/ViewModels/SchedulingViewModel.cs) ([SchedulingViewModel.vb](./VB/DbContextPerRequest/ViewModels/SchedulingViewModel.vb))
+- [SchedulingView.xaml](./CS/Shared/Views/SchedulingView.xaml) ([SchedulingView.xaml](./VB/Shared/Views/SchedulingView.xaml))
+- [SchedulingView.xaml.cs](./CS/Shared/Views/SchedulingView.xaml.cs) ([SchedulingView.xaml.vb](./VB/Shared/Views/SchedulingView.xaml.vb))
```

Comment:
Let's break down the list by project - otherwise it's hard to see what belong where. 

```suggestion
_Shared_ project:

- [SchedulingView.xaml](./CS/Shared/Views/SchedulingView.xaml) ([SchedulingView.xaml](./VB/Shared/Views/SchedulingView.xaml))
- [SchedulingView.xaml.cs](./CS/Shared/Views/SchedulingView.xaml.cs) ([SchedulingView.xaml.vb](./VB/Shared/Views/SchedulingView.xaml.vb))

_CommonDbContext_ project:

- [SchedulingViewModel.cs](./CS/CommonDbContext/ViewModels/SchedulingViewModel.cs) ([SchedulingViewModel.vb](./VB/CommonDbContext/ViewModels/SchedulingViewModel.vb))

_DbContextPerRequest_ project: 

- [SchedulingViewModel.cs](./CS/DbContextPerRequest/ViewModels/SchedulingViewModel.cs) ([SchedulingViewModel.vb](./VB/DbContextPerRequest/ViewModels/SchedulingViewModel.vb))
```